page/model.mjs

```diff
@@ -0,0 +1,29 @@
+import * as utils from "../utilities/shared.mjs"
+
+export async function findPageById(id = null) {
+    let page = null;
+  
+    if (!utils.validateProjectID(id)) {
```

Comment:
This validation is for a projectID, not a pageID. In this case, it isn't broken yet, but it is important to see that the object validation here is not honored.

Reply:
Updated this , please check.